modules/machine-user-infra-machines-iso.adoc

```diff
@@ -2,22 +2,52 @@
 //
 // * machine_management/user_infra/adding-bare-metal-compute-user-infra.adoc
 // * post_installation_configuration/node-tasks.adoc
+ifeval::["{context}" == "multi-architecture-configuration"]
+:multi:
+endif::[]
 
 :_content-type: PROCEDURE
 [id="machine-user-infra-machines-iso_{context}"]
-= Creating more {op-system} machines using an ISO image
+= Creating {op-system} machines using an ISO image
 
 You can create more {op-system-first} compute machines for your bare metal cluster by using an ISO image to create the machines.
 
 .Prerequisites
 
+ifdef::multi[]
+* You must have a HTTP or HTTPS server. 
+* You installed the OpenShift CLI (oc).
+endif::multi[]
+ifndef::multi[]
 * Obtain the URL of the Ignition config file for the compute machines for your cluster. You uploaded this file to your HTTP server during installation.
+endif::multi[]
 
 .Procedure
 
+ifdef::multi[]
+. You can extract the Ignition config file from the cluster with the following command:
++
+[source,terminal]
+----
+$ oc extract -n openshift-machine-api secret/worker-user-data-managed --keys=userData --to=- > worker.ign
+----
+
+. Upload the worker.ign Ignition config file you exported from your cluster to your HTTP server. Note the URLs of these files 
+
+. You can validate that the ignition files are available on the URLs. The following example gets the Ignition config files for the compute node: 
++
+[source,terminal]
+----
+$ curl -k http://<HTTP_server>/worker.ign 
+----
+
+. Download the latest {product-version} 64-bit ARM RHCOS ISO from the link:https://mirror.openshift.com/pub/openshift-v4/arm64/dependencies/rhcos/latest/[RHCOS mirrors page]. 
+endif::multi[]
+
 . Use the ISO file to install {op-system} on more compute machines. Use the same method that you used when you created machines before you installed the cluster:
 ** Burn the ISO image to a disk and boot it directly.
 ** Use ISO redirection with a LOM interface.
+//not sure if this part is needed for multi-arch 
```

Comment:
Yes @kelbrown20 . I believe LOM is short for 'Lights Out Management', so that does apply here. The text below about interrupting the live boot sequence applies as well.

Reply:
Awesome, will remove that comment then, thank ya!